### DISCUSSION TITLE
adding pytube (and other python modules)?

### DISCUSSION THREAD

**Discussion by tct123** [beeware/briefcase#1018] [Q&A]
Hello,
i would like to know, how to add pytube (and other modules) to my Briefcase/Toga-project.
I am trying to make an (android) YouTube-downloader. When I am trying to build that, it fails.

I am using a debian based linux (64 bit) with python3.10.

**Comment by mhsmith** [comment]
Pytube is a pure-Python package, so it should work fine on Android. Please post the full error message from your build.
